bindata/network/ovn-kubernetes/managed/common/004-config.yaml

```diff
@@ -136,13 +136,19 @@ data:
 {{- if (index . "V6JoinSubnet") }}
     v6-join-subnet="{{.V6JoinSubnet}}"
 {{- end }}
+{{- if (index . "V4InternalMasqueradeSubnet")}}
+    v4-internal-masquerade-subnet="{{.V4InternalMasqueradeSubnet}}"
+{{- end }}
+{{- if (index . "V6InternalMasqueradeSubnet")}}
+    v4-internal-masquerade-subnet="{{.V6InternalMasqueradeSubnet}}"
```

Comment:
Typo?  should this be `v6-internal-masquerade-subnet` here?

Reply:
Upstream is `v6-masquerade-subnet`?   Did we add `internal-` somewhere?

Reply:
good catch on the typo corrected,

Reply:
yes we added the internal name https://github.com/openshift/api/pull/1410/files#diff-2e98c189fdf696589c6d28124557b9cf96a329e82f790b4e61209351cc68648d